lock-api/src/main/java/com/palantir/lock/client/LockLeaseService.java

```diff
@@ -164,12 +166,14 @@ private Set<LeasedLockToken> refreshTokens(Set<LeasedLockToken> leasedTokens) {
             return leasedTokens;
         }
 
-        ConjureRefreshLocksResponse refreshLockResponse =
-                delegate.refreshLocks(ConjureRefreshLocksRequest.of(serverTokens(leasedTokens)));
+        ConjureRefreshLocksResponseV2 refreshLockResponse =
+                delegate.refreshLocksV2(ConjureRefreshLocksRequestV2.of(serverTokens(leasedTokens)));
         Lease lease = refreshLockResponse.getLease();
 
         Set<LeasedLockToken> refreshedTokens = leasedTokens.stream()
-                .filter(t -> refreshLockResponse.getRefreshedTokens().contains(t.serverToken()))
+                .filter(t -> refreshLockResponse
+                        .getRefreshedTokens()
+                        .contains(LockLeaseService.convertV1Token(t.serverToken())))
```

Comment:
Is there a reason why converting to v1 is more beneficial than just having LeasedLockToken use v2?

Reply:
Ah I see you actually mention this. This seems correct, but janky -- I'd prefer the full change, but I am also ok with leaving that for a follow-up PR

Reply:
Yeah, let's do that in a follow up, given this is time sensitive